LadybugTools_Engine/Python/src/ladybugtools_toolkit/bhom/__init__.py

```diff
@@ -1,6 +1,22 @@
 """Root for the bhom subpackage."""
 
-from .analytics import decorator_factory
-from .log import console_logger
+from pathlib import Path  # pylint: disable=E0401
 
-CONSOLE_LOGGER = console_logger()
+from win32api import HIWORD, LOWORD, GetFileVersionInfo
+
+BHOM_ASSEMBLIES_DIRECTORY = Path("C:/ProgramData/BHoM/Assemblies")
+BHOM_DIRECTORY = Path("C:/ProgramData/BHoM")
+BHOM_LOG_FOLDER = Path("C:/ProgramData/BHoM/Logs")
+
+PYTHON_CODE_DIRECTORY = Path("C:/ProgramData/BHoM/Extensions/PythonCode")
+PYTHON_ENVIRONMENTS_DIRECTORY = Path(
+    "C:/ProgramData/BHoM/Extensions/PythonEnvironments"
+)
```

Comment:
@Tom-Kingstone can you raise an issue about changing these from `C:` to a generic `ProgramData`. In C#, this can be done by using `System.Environment.GetFolderPath(System.Environment.SpecialFolder.CommonApplicationData)` where `System.Environment.GetFolderPath()` method gets the folder path for a given folder and `System.Environment.SpecialFolder.CommonApplicationData` is the enum for determining `ProgramData` over `AppData` or other common folders within the system.

I'd hope Python has a similar ability 😄 

FYI @tg359